platform/posix/transport/src/openssl_posix.c

```diff
@@ -125,6 +125,16 @@ static int setCredentials( SSL_CTX * pSslContext,
 static void setOptionalConfigurations( SSL * pSsl,
                                        const OpensslCredentials_t * pOpensslCredentials );
 
+/**
+ * @brief Converts the sockets wrapper statuses to openssl statuses.
```

Comment:
Nitpick suggestion as this only takes a single status value as input parameter, not multiple status_es_
```suggestion
 * @brief Converts the sockets wrapper status to openssl status.
```